packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -60,13 +60,13 @@ After installing ngrok, ensure that it's running by creating a "tunnel" into a l
 sample-app > ./ngrok http 8082
 ```
 
-or if you installed in your system path:
+Or if you installed in your system path:
```

Comment:
Could we put these three options in a table instead? The formatting here seems funky.

Reply:
Removed two options. I think there's enough info here, and users can consult ngrok docs if they're having issues.